src/main/resources/application.conf

```diff
@@ -1,8 +1,28 @@
 
 
-h2mem1 = {
-  url = "jdbc:h2:mem:test1"
-  driver = org.h2.Driver
-  connectionPool = disabled
-  keepAliveConnection = true
-}
\ No newline at end of file
+database {
+  config = h2mem1
+
+  h2mem1 = {
+    url = "jdbc:h2:mem:test1"
```

Comment:
Right. There was a request to try hard to make this done by end of sprint so @kshakir was just trying to move some stuff along while @cjllanwarne was away